Bugfix 1035 (#1261)
man-group/ArcticDB: cpp/arcticdb/CMakeLists.txt

    list(REMOVE_ITEM NEW_FOLLY_DEPS "/usr/local/opt/openssl/include" )
    set_target_properties(Folly::folly_deps PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${NEW_FOLLY_DEPS}")

    add_library(xxHash STATIC IMPORTED)
    set_target_properties(xxHash PROPERTIES IMPORTED_LOCATION ${xxHash_LIBRARY})

>     set_target_properties(xxHash PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${xxHash_INCLUDE_DIR})

Reviewer: Shouldn't this be done for conda packages and for wheels (i.e. independently from `ARCTICDB_USING_CONDA`)?

Author: `xxHash` isn't a target in the vcpkg build, so this won't work

# on their origin (i.e. vendored source or packages on conda-forge).
if(${ARCTICDB_USING_CONDA})

>     # We must use dynamic libraries, except for xxHash, which must be statically linked.

Reviewer: ```suggestion
    # We must use dynamic libraries, except for xxHash, which must be statically linked
    # since `XXH64_state_t` which is used ArcticDB is only available for static linkage.
```

Author: Rephrased

>     # In non-conda builds, we inline the functions so it does not need linking at all.

Reviewer: ```suggestion
    # For all builds but linux conda's, we inline the functions so that
    # xxhash does not need linking at all.
```

Author: Rephrased

    list (APPEND arcticdb_core_libraries
        pcre
        msgpack-c
        ${LMDB_LIBRARIES}
        xxHash

>     )

Reviewer: Was this change needed because of vcpkg's distribution?

Author: No, you can use either, I'd just removed it and then put it back with the alternative name. Reverted now.
